Sources/Common/DataModel/ITKHelper/index.js

```diff
@@ -0,0 +1,30 @@
+import vtkImageData from 'vtk.js/Sources/Common/DataModel/ImageData';
+import vtkDataArray from 'vtk.js/Sources/Common/Core/DataArray';
+
+/**
+ * Converts an itk.js image to a vtk.js image.
+ *
+ * Requires an itk.js image as input.
+ */
+function convertItkToVtkImage(itkImage, options) {
+  // create VTK image data
+  const imageData = vtkImageData.newInstance({
+    origin: itkImage.origin.slice(),
+    spacing: itkImage.spacing.slice(),
+  });
+  const scalars = vtkDataArray.newInstance({
```

Comment:
It would be good to add a `switch` on `itkImage.imageType.pixelType` per

  https://github.com/InsightSoftwareConsortium/itk-js/blob/master/src/PixelTypes.js

where `Scalar`, `RGB`, and `RGBA` we do this, and `default` to throwing an error for not implemented yet. Per:

  https://www.vtk.org/doc/nightly/html/classvtkDataSetAttributes.html#a156a8842bffd9139fa81193cca0d5bcb

Reply:
Moved to #582